bootstrap

```diff
@@ -10,7 +10,7 @@ buildroot="$(pwd)"/buildroot
 src="$buildroot"/src
 stages="$(pwd)"/stages
 
-crystal_stages="160"
+crystal_stages="162"
```

Comment:
It doesn't feel great to have to maintain this number.

Maybe we can just query the maximum value like this?
```suggestion
crystal_stages="$(grep -o -P '(?<=^stage )\d+' $0 | tail -n1)"
```

(Although it's still incorrect because stage 162 is duplicated.)

Reply:
I agree, it doesn't make sense to update that value.
What you have is good, I was thinking similar.
As far as the dupe stage, should it actually be 163 instead?

Reply:
> As far as the dupe stage, should it actually be 163 instead?

Yes, I'm pretty sure it should.